apps/coursition/src/app/media/page.tsx

```diff
@@ -116,6 +115,9 @@ export default function Index() {
             )}
           </div>
         )}
+        {status.value === 'done' && (
+          <TranscriptionResults raw={state.raw ?? ''} srt={state.srt ?? ''} vtt={state.vtt ?? ''} />
```

Comment:
**suggestion:** If some content is optional, it's better to just not pass it at all rather than empty string. And if it's undefined, just don't show the tab at all.